ts/api/backend.ts

```diff
@@ -226,8 +238,15 @@ export function BackendClient(
     response_decoder: getActivationStatusDefaultDecoder()
   };
 
+  const withBearerToken = <P extends { Bearer: string }, R>(
```

Comment:
@Undermaken with this wrapper we "remove" the `Bearer` parameter from the API calls as we use a constant value (`token`)